website/docs/sdk-reference/dart.md

```diff
@@ -418,8 +427,24 @@ final user = ConfigCatUser(identifier: '435170f4-8a8b-4b67-a723-505ac7cdea92');
 final settingValuesTargeting = await client.getAllValues(user);
 ```
 
-## Custom Cache
+## Cache
+
+The SDK caches the downloaded `config.json` only in memory by default. In case you have a Flutter application, you can use our [Flutter Preferences Cache](https://github.com/configcat/flutter-preferences-cache) for caching.  
```

Comment:
```suggestion
The SDK caches the downloaded `config.json` only in memory by default. In case you have a Flutter application, you can use the [Flutter Preferences Cache](https://github.com/configcat/flutter-preferences-cache) for caching.  
```